test/browser.multiapp.test.js

```diff
@@ -35,29 +36,42 @@ describe('browser support for multiple apps', function() {
     browserifyTestApps(apps, function(err, bundlePath) {
       if (err) return done(err);
 
-      var bundledApps = executeBundledApps(bundlePath, apps);
-      var app1 = bundledApps.defaultApp;
-      var app2 = bundledApps.browserApp2;
+      var bundledApps = executeBundledApps(bundlePath, apps, function(err) {
+        var app1 = bundledApps.defaultApp;
+        var app2 = bundledApps.browserApp2;
 
-      expect(app1.settings).to.have.property('custom-key', 'custom-value');
-      expect(Object.keys(app1.models)).to.include('Customer');
-      expect(Object.keys(app1.models)).to.not.include('Robot');
-      expect(app1.models.Customer.settings).to.have.property('_customized',
-        'Customer');
+        expect(app1.settings).to.have.property('custom-key', 'custom-value');
+        expect(Object.keys(app1.models)).to.include('Customer');
+        expect(Object.keys(app1.models)).to.not.include('Robot');
+        expect(app1.models.Customer.settings).to.have.property('_customized',
+          'Customer');
 
-      expect(Object.keys(app2.models)).to.include('Robot');
-      expect(Object.keys(app2.models)).to.not.include('Customer');
+        expect(Object.keys(app2.models)).to.include('Robot');
+        expect(Object.keys(app2.models)).to.not.include('Customer');
 
-      done();
+        done();
+      });
     });
   });
 });
 
+function addPlugins(b) {
+  var files = fs.readdirSync(path.join(__dirname, '../lib/plugins'));
+  files.forEach(function(f) {
+    b.require('../../lib/plugins/' + f,
+      { expose: './plugins/' + path.basename(f, '.js') });
```

Comment:
I would expect that `boot.compileToBrowserify` will handle inclusion of plugin sources, in the same way as it includes model scripts and boot scripts. What are the arguments for requiring LoopBack users to explicitly bundle all plugins themselves? Wouldn't it result in everybody copying four lines above, possibly with few tweaks in file locations?